cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -854,6 +926,21 @@ fn is_callee_save_baldrdash(r: RealReg) -> bool {
     }
 }
 
+fn is_callee_save_fastcall(r: RealReg) -> bool {
+    use regs::*;
+    match r.get_class() {
+        RegClass::I64 => match r.get_hw_encoding() as u8 {
+            ENC_RBX | ENC_RBP | ENC_RSI | ENC_RDI | ENC_R12 | ENC_R13 | ENC_R14 | ENC_R15 => true,
+            _ => false,
+        },
+        RegClass::V128 => match r.get_hw_encoding() as u8 {
+            6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 => true,
+            _ => false,
+        },
+        _ => unimplemented!(),
```

Comment:
More detail? `panic!("unable to save fastcall callee register for the register class: {}", r.get_class())`

Reply:
Done!